iron-image.html

```diff
@@ -105,9 +114,11 @@
 
   <template>
 
-    <img id="img" role="none" hidden$="[[_computeImageVisibility(sizing)]]">
-    <div id="placeholder" hidden$="[[_computePlaceholderVisibility(fade,loaded,preload)]]" class$="[[_computePlaceholderClassName(fade,loaded,preload)]]"></div>
-    <content></content>
+    <div id="backdrop" hidden$="[[_computeBackdropHidden(sizing)]]"></div>
+    <img id="img" role="none" hidden$="[[_computeImgHidden(sizing)]]">
```

Comment:
I buy that; all the accessibility information they get should come from the host since it has `role="img"`.